server_addon/create_ayon_addons.py

```diff
@@ -303,6 +314,41 @@ def main(create_zip=True, keep_source=False):
 
 
 if __name__ == "__main__":
-    create_zip = "--skip-zip" not in sys.argv
-    keep_sources = "--keep-sources" in sys.argv
-    main(create_zip, keep_sources)
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--skip-zip",
+        dest="skip_zip",
+        action="store_true",
+        help=(
+            "Skip zipping server package and create only"
+            " server folder structure."
+        )
+    )
+    parser.add_argument(
+        "--keep-sources",
+        dest="keep_sources",
+        action="store_true",
+        help=(
+            "Keep folder structure when server package is created."
+        )
+    )
+    parser.add_argument(
+        "-o", "--output",
+        dest="output_dir",
+        default=None,
+        help=(
+            "Directory path where package will be created"
+            " (Will be purged if already exists!)"
+        )
+    )
+    parser.add_argument(
+        "-c", "--dont-clear-output",
+        dest="clear_output_dir",
+        action="store_false",
+        help=(
+            "Clear output directory before creating packages."
+        )
+    )
+
+    args = parser.parse_args(sys.argv[1:])
+    main(args.output_dir, args.skip_zip, args.keep_sources, args.clear_output_dir)
```

Comment:
This is quite confusing - the argument is called `--dont-clear-output` but the variable name it goes into and the `main` argument `clear_output_dir` seems to be the exact opposite.

Shouldn't the flag thus be:
```
--clear-existing-output
```

Reply:
I was starting from the default behavior, without the arg, it was clear all by default. This should be flagging it so it is not clear it. @iLLiCiTiT was telling me he had a better idea of the functionality and this will not be needed anyway.

Reply:
I had, then I realized that the logic still should be available, so I've just reversed it. By default it does not remove output directory, only if `--clear-output-dir` is passed it it will.